SimCalorimetry/HGCalSimProducers/src/HGCDigitizer.cc

```diff
@@ -36,6 +40,20 @@ namespace {
     return geom->getGeometry(id)->getPosition().mag();
   }
 
+  int getCellThickness(const HGCalGeometry* geom, const DetId& detid ) {
+    const auto& topo     = geom->topology();
+    const auto& dddConst = topo.dddConstants();
+    uint32_t id(detid.rawId());
+    HGCalDetId hid(id);
+    int wafer = HGCalDetId(id).wafer();
+    int waferTypeL = dddConst.waferTypeL(wafer);
+    return waferTypeL;
+  }
+
+  int getCellThickness(const HcalGeometry* geom, const DetId& detid ) {
+    return 1;
```

Comment:
magic number should be made into a descriptive const, or ideally accessed from geometry/topology info somehow

Reply:
This I don’t know how to change. I copied from elsewhere as in https://github.com/amartelli/cmssw/blob/HGChits_ToAforPU_PR94X/SimCalorimetry/HGCalSimProducers/src/HGCDigitizerBase.cc#L13

Reply:
Hmm... okay. Something to standardize in the future.